torch/jit/_trace.py

```diff
@@ -202,12 +216,6 @@ def verify(model, args, loss_fn=torch.sum, devices=None):
     # should be possible to check if our execution actually obeyed the 'devices'
     # the user provided.
 
-    # TODO: Consider adding a utility function to torch.jit to test
-    # for this case
-    if not isinstance(model, torch._C.CompiledFunction):
-        raise TypeError(
-            "Cannot verify an uncompiled module.  Add @torch.jit.compile to compile it"
-        )
```

Comment:
This change seems unrelated to adding type annotations. I can't judge if this check is or isn't still needed, but it's a functional change - can you comment on why this is okay?

Reply:
I actually don't know if it's okay or not either, but I do know that `torch._C.CompiledFunction` doesn't exist. I searched the entire codebase for `CompiledFunction` and found nothing.

Reply:
Ah, it's not used at all currently, and the one test for it in `test/test_jit.py` says:
```
@unittest.skip("verify needs to be updated to work with GraphExecutors")
```

Which probably means you want to put back this code and slap a `# type: ignore` on it.

Reply:
Okay, makes sense.